src/Npgsql/NpgsqlServerStatus.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Npgsql.BackendMessages;
+using System.Collections.Concurrent;
+using System.Net.Sockets;
+
+namespace Npgsql
+{
+    class NpgsqlServerStatus
+    {
+        internal enum ServerType
+        {
+            Unknown,
+            Down,
+            Primary,
+            Secondary
+        }
+    }
```

Comment:
No need in so deeply hidden enum. Move the enumerable to namespace and remove the class.